devtools/conda-envs/test_env.yaml

```diff
@@ -30,11 +30,11 @@ dependencies:
   - openmmforcefields >=0.9.0
   - openmmtools >=0.23 # may need to sort out ambermini/ambertools/parmed dependencies
   - openmoltools # may need to sort out ambermini/ambertools/parmed dependencies (we don't want ambertools)
-  - parmed # may need to sort out ambermini/ambertools/parmed dependencies
+  - parmed
   - pdbfixer
   - pip
   - progressbar2
-  - pymbar <4
+  - pymbar
```

Comment:
Do we want to pin the pymbar version here as well or only in the conda-forge recipe?

Reply:
I rather do it in the conda-forge recipe since if we don't have a pin in the yaml we use for tests, it is really easy then in CI to make a matrix testing different versions or like the way we do it now, since there is no pin, cpu tests get pymbar 4, but in our GPU CI yaml, we use the extra build args to then pin pymbar<4. If we put it in the env yaml, we would get an error because micomamba doesn't know what to do when the yaml says >3 but the cli command says <4